ptf/tests/common/base_test.py

```diff
@@ -117,26 +117,10 @@ def get_controller_packet_metadata(p4info, meta_type, name):
                         return m
 
 
-def de_canonicalize_bytes(bitwidth: int, input: bytes):
-    """
-    This method adds a padding to the 'input' param.
-    Needed for bmv2 since it uses Canonical Bytestrings: this representation
-    trims the data to the lowest amount of bytes needed for that particular value
-    (e.g. 0x0 for PacketIn.ingress_port will be interpreted by Stratum bmv2 using 1 byte, instead of 9 bits,
-    as declared in header.p4)
-    :param bitwidth: the desired size of input.
-    :param input: the byte string to be padded.
-    :return: padded input with bytes such that: len(bin(input)) >= bitwidth.
-    """
-    if bitwidth <= 0:
-        raise ValueError("bitwidth must be a positive integer.")
-    if input is None:
-        raise ValueError("input cannot be of NoneType.")
-
-    byte_width = (
-        bitwidth + 7
-    ) // 8  # use integer division to avoid floating point rounding errors.
-    return input.rjust(byte_width, b"\0")  # right padding <-> BigEndian
+def canonicalized(byte_string):
```

Comment:
shorter and easier to pronounce
```suggestion
def canonical(byte_string):
```